src/splat-convert.ts

```diff
@@ -71,6 +71,7 @@ const convertPly = (convertData: ConvertEntry[]) => {
     const headerText = [
         `ply`,
         `format binary_little_endian 1.0`,
+        `comment generated by super-splat`,
```

Comment:
```suggestion
        `comment generated by supersplat`,
```

Reply:
I like the word to be written "SuperSplat" as it is in the titel of my browser-window. I took the hyphen-version only, because it exsted already for compressed ply